src/plugins/plugin.legend.js

```diff
@@ -102,18 +102,23 @@ function getBoxWidth(labelOpts, fontSize) {
 var Legend = Element.extend({
 
 	initialize: function(config) {
-		helpers.extend(this, config);
+		var me = this;
+		helpers.extend(me, config);
 
+		if (me.options && !me.align) {
+			// to maintain backward compatibility with existing default
+			me.options.align = me.isHorizontal() ? 'center' : 'start';
```

Comment:
I'm fine with `center` as the default going forward. We'd probably just want to mention it in the release notes

Reply:
+1 to default of `center`

Reply:
From an expected behaviour perspective I'm in consensus in that the default should be `center`.  Being unsure as to how closely this project adheres to semver, I opted to make this releasable as a minor by the strictest interpretation.  

Practically, I think releasing as a minor with a default change to `center` is perfectly reasonable with a mention in the release notes, as suggested by @etimberg.  It sounds like this is the preferred approach - I'll make the necessary changes in an upcoming commit.

Reply:
> Being unsure as to how closely this project adheres to semver ...

We are certainly not strict enough, but should we consider all slight visual changes as "breaking changes"? I tend to think we shouldn't because users are still able to use / interact with the chart as before and without issues, while we are making the API more consistent / flexible / robust. But I'm open to consider a fallback to `start` if you guys think it's a breaking change.

I also agree, this PR should go in the next minor release.

> I opted to make this releasable as a minor by the strictest interpretation

That's definitely the best approach!

Reply:
I'm fine with `center` as the default, too.